### PR TITLE
Use config object for site page titles

### PR DESCRIPTION
**Pull request by jseppi**
here you go, @line47. ref #1429.

screen shot 2018-01-05 at 9 35 13 am

### CODE REVIEW

**Comment by line47** [comment]
💯 Awesome, thanks @jseppi !

File: frontend/components/siteContainer.js [hunk 1]
```diff
@@ -39,9 +39,37 @@ const defaultProps = {
...
+  {
+    display: 'GitHub branches',
+    route: 'branches',
+  },
+  {
+    display: 'Published files',
+    route: 'published',
+  },
+  {
+    display: 'Users',
```
Comment (line47) [inline frontend/components/siteContainer.js]: Change "Users" to "Collaborators"

File: frontend/components/siteContainer.js [hunk 2]
```diff
@@ -39,9 +39,37 @@ const defaultProps = {
...
+    route: 'users',
+  },
+  {
+    display: 'Settings',
```
Comment (line47) [inline frontend/components/siteContainer.js]: Update "Settings" to "Site Settings"

**Comment by jseppi** [comment]
@line47: updated!

**Pull request merged**